Assignment5/Assignment5_QNo3/module_instance_method.rb

```diff
@@ -0,0 +1,10 @@
+# => 3. Write a program to define a method inside a module and then use the
+# => module in such a way that the method can be accessible as an instance
+# => method of that class.
+module ModuleDemoInstanceMethod
+  # => define common method
+  def method_instance_behave
+    puts "The addition of two numbers 9 + 5 = 14"
+  end
+end
```

Comment:
@HrisikeshSinghaChowdhury You need to write program to use this module inside a class to access the module method as an instance method

Reply:
there is another file int this folder where i implemented the same
Name of file => SolnAssignmentNo5_QNo3.rb